### ISSUE TITLE
SourceChangeWarning & WeightDropout error

### ISSUE THREAD

**Issue by TC1997** [paperswithcode/axcell#10]
Hi！
I am reproducing the results and encountered two problems as follows:
1.SourceChangeWarning
When using ResultsExtractor in evaluation.ipynb, the error was raised: **_453: SourceChangeWarning: source code of class 'torch.nn.modules.loss.BCEWithLogitsLoss' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes._**
Could you please give me some advice on how to retrieve the original source?

2.WeightDropout
When extracting the results in evaluation.ipynb, an error was raised: **_AttributeError: 'WeightDropout' object has no attribute 'idxs'_**

Looking forward to your kind response:) Thank you very much!

**Comment by mkardas** [comment]
Hi, I've updated `environmet.yml`. Can you recreate your conda environment and check if the issue is still there?

**Comment by TC1997** [comment]
Thanks for updating! There are still SourceChangeWarnings but no errors now and I can reproduce the results.
